### PR TITLE
Add Lint commit hook and prettify all files

### PR DESCRIPTION
**Pull request by patsimm**
Prettify all files and add ng lint commit hook

### CODE REVIEW

File: src/app/services/geo.service.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ export class GeoService {
   getLocations(radius: number, coords: Array<number>) {
     const query: GeoQuery = this.locations.near({
       center: new firestore.GeoPoint(coords[0], coords[1]),
-      radius: radius,
+      radius,
```
Comment (darkblackside) [inline src/app/services/geo.service.ts]: Is this working properly?

File: src/index.html [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
 <html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Lokalkauf</title>
+    <base href="/" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
```
Comment (darkblackside) [inline src/index.html]: I think this is a strange formatting rule. Can we disable this?

File: src/app/services/geo.service.ts (hunk 1 above)
Comment (patsimm) [reply src/app/services/geo.service.ts]: yes, this is not a problem and will work as it should. https://eslint.org/docs/rules/object-shorthand

File: src/index.html (hunk 2 above)
Comment (patsimm) [reply src/index.html]: what exactly do you mean? We could increase the max-line-length so it doesn't break in that case. I think thats whats causing it.

**Comment by darkblackside** [comment]
Can you resolve the conflicts?

**Pull request merged**